hibike/hibike_process.py

```diff
@@ -75,8 +75,16 @@ def recv_subscription_requests(conn, uid_queue, stop_event):
     thread_list = []
     event_list = []
     read_queues = []
+
     for conn in serial_conns:
-        hm.send(conn, hm.make_ping())
+        old_timeout = conn.write_timeout
+        conn.write_timeout = IDENTIFY_TIMEOUT
```

Comment:
The definition of `IDENTIFY_TIMEOUT` shouldnt be hiding halfway through this file.